apps/omg_status/lib/status.ex

```diff
@@ -23,53 +23,23 @@ defmodule OMG.Status do
   def start(_type, _args) do
     import Supervisor.Spec, warn: false
 
-    Enum.map(vm_metrics(), fn {name, invoke} ->
-      Recorder.prepare_child(%Recorder{
-        name: name,
-        fn: invoke,
-        reporter: &Appsignal.set_gauge/3
-      })
-    end)
-    |> Supervisor.start_link(strategy: :one_for_one, name: Status.Supervisor)
+    children =
+      case is_enabled?() do
```

Comment:
case with booleans looks strange to me :)
how about
```
Application.put_env(:vmstats, :sink, Status.Metric.Recorder)
children = if is_enabled?(), do: [Recorder.prepare_child()], else: []

Reply:
Haha, yeah ... I come from erlang where everything is pattern matching